Onboarding/sbm-send-pregnancy-message.md

```diff
@@ -88,20 +105,81 @@ card GetMessage when date == today(), then: SendMessage do
   page = page.body
 end
 
+card GetMessage when schedule_date != today(), then: CalculateTimestamp do
+  page = page + 1
+end
```

Comment:
I'm not sure that adding this does anything? What was the reasoning here?

Reply:
This was done for the scheduled messages, so we loop through the messages to see if we have one foe today , if not then we check the next message